src/components/form-helpers/form-helpers.js

```diff
@@ -320,9 +321,17 @@ const _PhoneFieldItem = ({
       autoPlaceholder: 'aggressive',
       preferredCountries: ['ar', 'mx', 'co', 'es', 'ec', 'cl', 'pe', 'us'],
       initialCountry: 'auto',
-      geoIpLookup: (callback) => {
-        // TODO: determine current country using geolocation
-        callback('ar');
+      geoIpLookup: async (success, failure) => {
+        // TODO: read cache before request
+        try {
+          const response = await axios.get('https://ipinfo.io/json');
+          var countryCode = response && response.country ? response.country : '';
```

Comment:
Also, if I am not wrong, we could take advantage of new ES features (we should verify the generated code):

```suggestion
          const countryCode = response?.country ?? '';
```

Reply:
But... why would you want an empty string in `countryCode` 😕?

Reply:
In this case I think that we should be return AR by default